ckan-client-guide/README.md

```diff
@@ -4,113 +4,523 @@ Guide to interacting with [CKAN](/ckan) for power users such as data scientists,
 
 This guide is about adding and managing data in CKAN programmatically and it assumes:
 
-* You are familiar with key concepts like metadata, data etc.
-* You are working programmatically e.g. in one of Python, R, JavaScript, etc.
+* You are familiar with key concepts like metadata, data, etc.
+* You are working programmatically with a programming language such as Python, JavaScript or R (_coming soon_).
 
 ## Frictionless Formats
 
-Clients use [Frictionless formats][f11s] by default for describing dataset and resource objects passed to client methods. Internally, we then use the a CKAN<=>Frictionless Mapper ([js][c2f], [python][c2fpy] to convert to CKAN formats before calling the API. Thus, you can use Frictionless Formats by default with the client. (As CKAN moves to Frictionless to default this will gradually become unnecessary).
+Clients use [Frictionless formats](https://specs.frictionlessdata.io/) by default for describing dataset and resource objects passed to client methods. Internally, we then use the a *CKAN <=> Frictionless Mapper* (both [in JavaScript]( https://github.com/datopian/frictionless-ckan-mapper-js ) and [in Python](https://github.com/frictionlessdata/frictionless-ckan-mapper)) to convert objects to CKAN formats before calling the API. **Thus, you can use _Frictionless Formats_ by default with the client**.
```

Comment:
I prefer out of flow links, at least for substantive items that might be repeated i.e.

```
... [Frictionless formats][f11s] ....

[f11s]: ...
```

Will merge here and worth noting for the future.